### DISCUSSION TITLE
How to split a long diagnostic line from LSP

### DISCUSSION THREAD

**Discussion by Zeta611** [LazyVim/LazyVim#2368] [Q&A] [answered]
How do I split a long diagnostic line from LSP? I'm using `ocamllsp`.

<img width="868" alt="Screenshot 2024-01-17 at 21 45 25" src="https://github.com/LazyVim/LazyVim/assets/9553691/d0b3d685-f1b7-41cf-a55f-0b2bf5608cae">

**Comment by dpetka2001** [comment, marked as answer]
Does using `<leader>cd` help you as an alternative way? Otherwise, I believe you would have to change [vim.diagnostic.config](https://neovim.io/doc/user/diagnostic.html#vim.diagnostic.config%28%29) to format the `virtual_text` how you want, but unfortunately I don't know how to do that to split messages in multilines. Also, if you press `<leader>cd` a second time it will take your cursor into the floating window with the message and you're able to use normal vim motions to move around and copy things if you'd like.

**Reply by Zeta611** [reply]
Awesome! This works for me :) Thank you.
